programs/bubblegum/program/src/processor/finalize_tree_with_root.rs

```diff
@@ -169,7 +177,9 @@ pub(crate) fn check_stake<'info>(
         BubblegumError::StakingVoterDiscriminatorMismatch
     );
 
-    let voter: Voter = *bytemuck::from_bytes(&(*voter_bytes.borrow())[8..]);
+    // The Voter has a rather big structure. If we try to deserialize it on the stack it will be overflowed. Therefore, Box is used to move bytes that have been borrowed to the Heap.
+    let voter_bytes = Box::new(voter_bytes.borrow());
+    let voter: &Voter = bytemuck::from_bytes(&voter_bytes[8..]);
```

Comment:
Perhaps, it would be more clear to make `DISCRIMINATOR.len()` or something like that instead of 8?